### PR TITLE
Add support for `include_inherited` and `progress_bar` in series values fetching

### PR DESCRIPTION
**Pull request by Raalsky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -464,10 +464,11 @@ By default, the maximum number of workers is 10. You can change this number by s
...
 
-| Name          | Type        | Default | Description                               |
-|---------------|-------------|---------|-------------------------------------------|
-| `paths`       | `List[str]`, required | None      | List of paths to prefetch to the internal cache.      |
-| `use_threads` | `bool`, optional      | `False` | Whether to use threads to fetch the data. |
+| Name                | Type                  | Default | Description                                                         |
+|---------------------|-----------------------|---------|---------------------------------------------------------------------|
+| `paths`             | `List[str]`, required | `None`  | List of paths to prefetch to the internal cache.                    |
+| `use_threads`       | `bool`, optional      | `False` | Whether to use threads to fetch the data.                           |
+| `include_inherited` | `bool`                | `True`  | Whether the fetched data should include the values from parent runs |
```
Comment (szaganek) [inline README.md:471]:
```suggestion
| `include_inherited` | `bool`, optional                | `True`  | Whether the fetched data should include values from the parent runs. |
```

File: README.md [hunk 2]
```diff
@@ -546,9 +547,10 @@ Retrieves all series values either from the internal cache (see [`prefetch_serie
 
 __Parameters:__
 
-| Name                | Type   | Default | Description                                                  |
-|---------------------|--------|---------|--------------------------------------------------------------|
-| `include_timestamp` | `bool` | True    | Whether the fetched data should include the timestamp field. |
+| Name                | Type   | Default | Description                                                         |
+|---------------------|--------|---------|---------------------------------------------------------------------|
+| `include_timestamp` | `bool` | `True`  | Whether the fetched data should include the timestamp field.        |
+| `include_inherited` | `bool` | `True`  | Whether the fetched data should include the values from parent runs |
```
Comment (szaganek) [inline README.md:553]:
```suggestion
| `include_inherited` | `bool` | `True`  | Whether the fetched data should include values from the parent runs. |
```

File: src/neptune_fetcher/fetchable.py [hunk 3]
```diff
@@ -121,23 +122,50 @@ def fetch(self) -> None:
...
+        include_timestamp: bool = True,
+        include_inherited: bool = True,
+        progress_bar: "ProgressBarType" = None,
+    ) -> "DataFrame":
+        """
+        Retrieves all series values either from the internal cache (see `prefetch_series_values()`) or from the API.
+
+        Args:
+            include_timestamp: Whether the fetched data should include the timestamp field.
+            include_inherited: Whether the fetched data should include the values from parent runs.
```
Comment (szaganek) [inline src/neptune_fetcher/fetchable.py:156]:
```suggestion
            include_inherited: Whether the fetched data should include values from the parent runs.
```

File: src/neptune_fetcher/read_only_run.py [hunk 4]
```diff
@@ -127,6 +131,7 @@ def prefetch_series_values(
             use_threads: If True, fetching is done concurrently.
             progress_bar: Set to `False` to disable the download progress bar,
                 or pass a `ProgressBarCallback` class to use your own progress bar callback.
+            include_inherited: If `False` values from parent runs will not be included. `True` at default.
```
Comment (szaganek) [inline src/neptune_fetcher/read_only_run.py:134]:
```suggestion
            include_inherited: If `False`, values from the parent runs will not be included.
```

File: src/neptune_fetcher/cache.py [hunk 5]
```diff
@@ -57,18 +57,35 @@ def prefetch(self, paths: List[str]) -> None:
...
+                fetch_values_concurrently(
+                    partial(self._fetch_single_series_values, include_inherited=include_inherited),
+                    paths=paths,
+                    progress_bar=progress_bar,
+                )
             else:
-                fetch_values_sequentially(self._fetch_single_series_values, paths, progress_bar)
-
-    def _fetch_single_series_values(self, path: str, progress_bar: ProgressBarCallback) -> None:
+                fetch_values_sequentially(
```
Comment (kgodlewski) [inline src/neptune_fetcher/cache.py:77]: Indentation 1 level too deep?

**Review by kgodlewski — APPROVED** [review]

**Pull request merged**